projects/use/useop-qwt-config.cmake

```diff
@@ -0,0 +1,28 @@
+# Get the path to the overseer pro install
+get_filename_component(OP_ROOTDIR ${CMAKE_CURRENT_LIST_DIR}/../.. ABSOLUTE)
+get_filename_component(OP_ROOTDIR ${OP_ROOTDIR} ABSOLUTE)
+
+unset(QWT_INCLUDE_DIR CACHE)
+unset(QWT_LIBRARY_DIR CACHE)
+
+set(QWT_INCLUDE_DIR ${OP_ROOTDIR}/include/qwt)
+set(QWT_LIB_DIR ${OP_ROOTDIR}/lib)
+
+# determine OP_BUILD_STATIC
+include(${CMAKE_CURRENT_LIST_DIR}/opopts.cmake)
+
+string(TOLOWER ${CMAKE_BUILD_TYPE} build_type)
+
+if(WIN32)  
+  if(${build_type} STREQUAL debug)
+    set(QWT_LIB ${QWT_LIB_DIR}/qwtd.lib)
+  else()
+    set(QWT_LIB ${QWT_LIB_DIR}/qwt.lib)
+  endif()
+else()
+  if(OP_BUILD_STATIC)
```

Comment:
When it's dynamic you just add qwt to the link libraries (similar to how Qt is Qt::Widgets) then it knows to find the dynamic library.  My thought was if we've built it static then we'll define this cmake variable and qwt becomes ${QWT_LIB} in the link line.  I've made it so it will build static or dynamic.  Qwt is LGPL but they have a static link exception so you don't have to provide source code if you statically link (see http://qwt.sourceforge.net/qwtlicense.html)